tests/sample-data/profile-service/profile.json

```diff
@@ -1,32 +1,33 @@
-{
-  "gender":
+[
   { "name": "gender",
     "value": "TEST MALE",
     "sources": [],
     "verifiers": []
   },
-  "nationality":
   {
     "name": "nationality",
     "value": "GBR",
     "sources": [],
     "verifiers": []
   },
-  "phone_number":
   {
     "name": "phone_number",
     "value": "+447474747474",
     "sources": [],
     "verifiers": []
   },
-  "given_names":
+  {
+    "name": "date_of_birth",
+    "value": "2080-09-11T10:05:08.000Z",
```

Comment:
Not sure that `value` here would be in that format? I was under the impression that it would be like `YYYY-MM-DD`

Reply:
Yeah, good shout!
Although, that `profile.json` fixture is a bit odd, it represents the decoded attributes list, where some attributes have been converted to non JSON type (aka class or type like Date), hence we need to revive them... and therefore, the value here being a date in ISO format is actually kinda valid, as it will be revived as a Date... but both are valid overall.

Reply:
Yeah might just be worth keeping at as `YYYY-MM-DD` if that is what the API returns, just for consistency.

Reply:
Updated